server/api/router.go

```diff
@@ -35,6 +36,7 @@ func createRouter(svr *server.Server) *mux.Router {
 	router.Handle("/api/v1/events", newEventsHandler(svr, rd)).Methods("GET")
 	router.Handle("/api/v1/feed", newFeedHandler(svr, rd)).Methods("GET")
 	router.Handle("/api/v1/history/operators", newHistoryOperatorHandler(svr, rd)).Methods("GET")
+	router.Handle("/api/v1/metrics", prometheus.Handler()).Methods("GET")
```

Comment:
`/metrics` is the default http path in github.com repo example.
Do you think it is better?